db/migrate/20170827091406_change_container_quota_items_columns_to_float.rb

```diff
@@ -0,0 +1,36 @@
+class ChangeContainerQuotaItemsColumnsToFloat < ActiveRecord::Migration[5.0]
+  class ContainerQuotaItem < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled # disable STI
+  end
+
+  def up
+    add_column :container_quota_items, :quota_desired_float, :float
+    add_column :container_quota_items, :quota_enforced_float, :float
+    add_column :container_quota_items, :quota_observed_float, :float
+
+    ContainerQuotaItem.all.each do |item|
+      item.quota_desired_float = parse_quantity(item.quota_desired)
+      item.quota_enforced_float = parse_quantity(item.quota_enforced)
+      item.quota_observed_float = parse_quantity(item.quota_observed)
+      item.save
+    end
+
+    remove_column :container_quota_items, :quota_desired
+    remove_column :container_quota_items, :quota_enforced
+    remove_column :container_quota_items, :quota_observed
+
+    rename_column :container_quota_items, :quota_desired_float, :quota_desired
+    rename_column :container_quota_items, :quota_enforced_float, :quota_enforced
+    rename_column :container_quota_items, :quota_observed_float, :quota_observed
```

Comment:
I didn't find a way to rename the columns and convert the values without adding the new columns first and then removing the old ones.